packages/mobile/src/components/lineup-tile/LineupTileStats.tsx

```diff
@@ -32,14 +41,12 @@ const formatPlayCount = (playCount?: number) => {
   return `${formatCount(playCount)} ${suffix}`
 }
 
-const useStyles = makeStyles(() => ({
+const useStyles = makeStyles(({ spacing, palette }) => ({
   stats: {
     flexDirection: 'row',
-    flex: 0,
     alignItems: 'stretch',
-    paddingVertical: 2,
-    marginRight: 10,
-    height: 26
+    paddingVertical: spacing(2),
+    marginHorizontal: spacing(2.5)
```

Comment:
I had thought the design had everything aligned to grid units. Why the 2.5 spacing here?